js/weechat.js

```diff
@@ -290,7 +290,7 @@
                     var ret = {};
                     var optionCode = parseInt(m[1]);
 
-                    if (optionCode > 43) {
```

Comment:
(credit squirrel on IRC) 43 was actually the length, but for some reason using chat_day_change didn't work until I made this change. But then I tested it without this change just now and it works. I dunno what's up. In any case, it works and is a little bit more descriptive.